libs/localization/src/lib/en.json

```diff
@@ -5,7 +5,9 @@
     "identityFeature": "Identity feature",
     "infoTitle": "Additional information",
     "spinButton": "Spin",
-    "editButton": "Edit"
+    "editButton": "Edit",
+    "decreaseValue": "Decrease value",
```

Comment:
not sure if where the original strings came from, but should we keep the "by one" clause ?

Reply:
Few of the increase and decrease is based on value in data. So for them we can't use 'by 1'. I updated to keep by one for [FeatureConfigurationFlyout.tsx](https://github.com/microsoft/responsible-ai-toolbox/pull/1806/files#diff-fd9c3511dddd161b6dec541d5d8f502602609085885647f7f84f613bc3e46e65) which had it previously.